DATA/SHIPS/loadouts.ini

```diff
@@ -1,6 +1,32 @@
+;; Newplayer Starblazer loadout
+[Loadout]
+nickname = ge_fighter3_start
+equip = ge_s_scanner_01
+equip = ge_s_tractor_01
+equip = cv_engine_fighter_light01, HpEngine01
+equip = cv_shield_graviton_light_std01, HpShield01
+equip = cv_power_light_std01, DpEngine01
+equip = cv_gun_laser_light01, HpWeapon01
+equip = cv_gun_laser_light01, HpWeapon02
+equip = cv_gun_laser_light01, HpWeapon03
+equip = cv_gun_laser_light01, HpWeapon04
+equip = ge_s_thruster_01, HpThruster01
+equip = LargeWhiteSpecial, HpHeadlight
+equip = FastSmallRed, HpRunningLight01
+equip = SlowSmallOrange, HpRunningLight01
+equip = SlowSmallOrange, HpRunningLight02
+equip = SlowSmallOrange, HpRunningLight05
+equip = SlowSmallOrange, HpRunningLight06
+equip = SlowSmallOrange, HpRunningLight07
+equip = DockingLightRedSmall, HpDockLight01
+equip = DockingLightRedSmall, HpDockLight02
+equip = contrail01, HpContrail01
+equip = contrail01, HpContrail02
+
+;; Newplayer CSV loadout (currently deprecated)

```

Comment:
We do not need deprecated code. Delete it.

Reply:
Done.